File: code/7.py
import math
def isprime(i):
	test = 2
	prime = 1
	while test<=math.sqrt(i):
		if i%test == 0:
			prime = 0
		test += 1
	return prime

num = 2
counter = 0
while counter<10000:
	if isprime(num) == 1:
		counter += 1
		if counter == 10001:
			break
	if counter == 10001:
		break
	num += 1
print(num)

File: code/7.cc
#include <iostream>
#include <cmath>
using namespace std;

bool isprime(long& i);

int main(){
	long num = 2;
	int counter = 0;
	while(counter < 100000){
		if(isprime(num)){
			counter++;
			if(counter == 10001) break;
		}
		if(counter == 10001) break;
		num++;
	}
	cout<<num<<endl;
	return 0;
}

bool isprime(long& i){
	long test = 2;
	bool prime = true;
	while(test <= sqrt(i)){
		if(i%test == 0)
			prime = false;
		test++;
	}
	return prime;
}
